src/codec/framed_read.rs

```diff
@@ -346,9 +350,9 @@ where
                 None => return Poll::Ready(None),
             };
 
-            tracing::trace!("poll; bytes={}B", bytes.len());
+            tracing::trace!(read.bytes = bytes.len());
             if let Some(frame) = self.decode_frame(bytes)? {
-                tracing::debug!("received; frame={:?}", frame);
+                tracing::debug!(?frame, "received;");
```

Comment:
What's the difference with having some lines where the message is first, and other lines where the values are first?

Reply:
`frame` is recorded as a key/value pair where the key is `frame` and the value is the `fmt::Debug` output of `frame`.

Reply:
Yea, I just noticed some are like this one, `debug!(?thing, "message")`, and others are `debug!("message", ?thing)`. If we can do either, we should probably be consistent (and if I can vote, I like the message first, since that tells me "why" there's a message).